{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2018-11760",
    "ASSIGNER": "security@apache.org"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "NVD-CWE-noinfo"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "http://www.securityfocus.com/bid/106786",
        "name": "106786",
        "refsource": "BID",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.apache.org/thread.html/a86ee93d07b6f61b82b61a28049aed311f5cc9420d26cc95f1a9de7b%40%3Cuser.spark.apache.org%3E",
        "name": "[user@spark.apache.org] 20190128 [SECURITY] CVE-2018-11760: Apache Spark local privilege escalation vulnerability",
        "refsource": "",
        "tags": []
      },
      {
        "url": "https://lists.apache.org/thread.html/6d015e56b3a3da968f86e0b6acc69f17ecc16b499389e12d8255bf6e%40%3Ccommits.spark.apache.org%3E",
        "name": "[spark-commits] 20190806 [spark-website] branch asf-site updated: CVE-2019-10099",
        "refsource": "",
        "tags": []
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "When using PySpark , it's possible for a different local user to connect to the Spark application and impersonate the user running the Spark application. This affects versions 1.x, 2.0.x, 2.1.x, 2.2.0 to 2.2.2, and 2.3.0 to 2.3.1."
      }
    ]
  }
}